includes/rest-api/class-sensei-rest-api-setup-wizard-controller.php

```diff
@@ -77,6 +79,15 @@ public function can_user_access_rest_api() {
 		return current_user_can( 'manage_sensei' );
 	}
 
+	/**
+	 * Check user permission for install plugins.
+	 *
+	 * @return bool Whether the user can install plugins.
+	 */
+	public function can_user_install_plugins() {
```

Comment:
Should we handle this more gracefully, and skip the install step on the frontend if the user has no permission to install plugins? 
(Probably in a future iteration.)

Reply:
I'll let this question to @donnapep :)

The only bad point that I see is that we don't track the plugins that the user would like to install.

Reply:
When I tested this, I got the following notice:
![Screen Shot 2020-05-30 at 10 44 46 AM](https://user-images.githubusercontent.com/1190420/83331208-a9f32380-a262-11ea-8a12-3a8e1173b993.jpg)

Given that all admins can install plugins, and by default we only give the `manage_sensei` capability to admins, this doesn't seem like something worth worrying about right now. Especially since it's handled somewhat gracefully.

Reply:
The most common case where an admin might not have permission is multi-site installs, where plugins can only be installed network-wide, AFAIK.

Reply:
Thinking about, another point is that in these cases the user probably won't see the setup wizard because they won't be able to install the Sensei plugin.